WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -229,19 +229,20 @@
         [
           t('.events'),
           true,
-          @competition.events.select(&:official?),
-          Event.all_official,
+          @competition.event_picker_events,
+          Event.official,
         ],
         [
           t('.unoff_events'),
           false, # Only show deprecated events if this competition has any.
-          @competition.events.select(&:deprecated?),
-          Event.all_deprecated,
+          @competition.events.deprecated,
```

Comment:
I guess these should be `@competition.event_picker_events.deprecated` (with the current implementation).